courses/beginner/10.md

```diff
@@ -12,30 +12,23 @@ In addition to its own controls, an InSpec profile can bring in the controls fro
 
 When a profile includes controls from another profile, it is usually referred to as a “meta profile” or a “profile overlay.” Those of us with Chef background sometimes call it a “wrapper profile”.
```

Comment:
Replace the opening with:
In addition to its own controls, an InSpec profile can "inherit" controls from one or more other InSpec profiles.

When a profile depends on controls from other profiles, it can be referred to as an "overlay" or "wrapper" profile. We'll use the term overlay profile in this section.

An overlay can include all, select specific, skip some, or modify controls.

Reply:
Done!